plugins/topology/README.md

```diff
@@ -127,6 +127,7 @@ The following permission must be granted to the [`ClusterRole`](https://backstag
           - kubevirt.io
         resources:
           - virtualmachines
+          - virtualmachineinstances
```

Comment:
isn't this change in main already as https://github.com/janus-idp/backstage-plugins/pull/2176/files is merged? 

pls rebase once